### ISSUE TITLE
Production environment

### ISSUE THREAD

**Issue by jhe10709** [Kodkollektivet/kodkollektivet.se#26]
I think we should have a discussion on where and how the production environment should work. My idea is that we are using a Digital Ocean server, this is a standard GNU/Linux installation and we have to fix everything. This is good if we have people interested in the admin parts. If we dont have people interested in that, i guess something like AWS och other platform as a service provider is good.

**Comment by megamos** [comment]
+1 (I fixed my Digital Ocean credits yesterday and set up a debian server (droplet as they call it))
Cause KK don't have it's own account on DO I guess?
Dind't have time to setup the dev env, but will have it fixed soon. Or should we go with an other distro?
If you gimmmi a public key I'll give U access.

And yes, lets have a discussion :D
Over coffee in a cozy place, skype or some geeky advanced encrypted communication tool.
All works for me, at most days n most times. How about tomorrow or monday after last lecture, >3pm.

**Comment by jherrlin** [comment]
No, at the moment Kodkollektivet dont have any DO account. But i guess we need to fix that later on. 

I would like to see a workflow where we push to GitHub and those commits get put i production with the help of https://travis-ci.com/ . Then we dont need to access the server to get the new code into production.
